src/components/text-input/TextInput.js

```diff
@@ -37,6 +38,7 @@ type Props = {
 
 const TextInput = ({
     className = '',
+    id,
```

Comment:
Is there a strong need for `id` to be passed in or can we just generate a unique id every time?

Reply:
probably not other than to prevent breaking existing implementations

Reply:
removing `htmlFor` and relying on just nesting everything removes the need for this